examples/get_stats/main.go

```diff
@@ -53,4 +55,29 @@ func main() {
 	}
 	log.Printf("Now printing result:\n")
 	log.Printf("%s", r.GetPayload())
+
+	// Large payload
+	file, err := os.Open("geos.txt")
```

Comment:
Do you need to check this in? Or have a script to generate it, or something.

Reply:
In this cl. The github ui is hard to use...